src/http/multipart.rs

```diff
@@ -85,7 +85,8 @@ pub(super) async fn receive_batch_multipart(
             _ => {
                 if let Some(name) = field.name().map(ToString::to_string) {
                     if let Some(filename) = field.file_name().map(ToString::to_string) {
-                        let content_type = field.content_type().map(|mime| mime.to_string());
+                        let content_type =
+                            field.content_type().map(std::string::ToString::to_string);
```

Comment:
`ToString` is in the prelude, so don't have `std::string::`; this applies to all the other uses of that as well.